third_party/riscv-compliance/BUILD.riscv-compliance.bazel

```diff
@@ -0,0 +1,97 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+exports_files(glob(["**/*"]))
+
+genrule(
```

Comment:
Do the two genrules in this file just emit these headers as-is? (no sed/awk/etc?).

Would these be better as files supplied in a patch?

Reply:
This is intentional explicitly so that we do not need to supply the files as a patch, since those are annoying to update. I'd like to stick to a model where as little code is behind the patch boundary as possible because that has burned me repeatedly with the current vendor script.